### PR TITLE
[Guide] Using Netlify CMS with NextJS

### PR DESCRIPTION
**Pull request by steffenz**
**Summary**
Another guide for #2092. 
I went with a slightly different approach to make this guide as simple as possible to "code along" with. Hopefully, my list of "internet cats" won't offend anybody 😂
I didn't want to depend on any boilerplate code, nor use complex configuration (like remarkable for parsing). I do however understand if you want to do things differently. 
This is my first PR ever, so I'm sorry in advance if I messed up somewhere!

I used some of the other guides as inspiration and copied the "Enable Identity and Git Gateway" part directly from the Gatsby documentation.

### CODE REVIEW

File: website/content/docs/nextjs.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        - label: 'Cats'
+          name: "cats"
+          widget: list
+          fields: 
+            - { label: "Name", name: "name", widget: "string"}
+            - { label: "Description", name: "description", widget: "text"}
+```
+
+Awesome! Netlify CMS should now be available at ```localhost:3000/static/admin/index.html```.
+Unfortunatly we can't edit our content just yet. First we need to move our code into a git repository, and create a new Netlify site.
```
Comment (talves) [inline website/content/docs/nextjs.md]: Unfortunately

Comment (steffenz) [reply website/content/docs/nextjs.md]: Great, thanks for your feedback! I just fixed the typo 😁

**Comment by erquhart** [comment]
So pumped to see this @steffenz!

@verythorough any chance you'd have time to review this?

**Comment by verythorough** [comment]
I'm pretty swamped right now, but I'll try to take a look later in the week.

**Comment by erquhart** [comment]
@steffenz is this still in progress? If it's ready to go, drop the WIP in the title and we'll get this moved forward.

**Comment by steffenz** [comment]
> @steffenz is this still in progress? If it's ready to go, drop the WIP in the title and we'll get this moved forward.

I think this is pretty much done. I removed the [WIP] tag now. I guess there's always room for improvements, but as a first version I think it's ready to go 😄

**Comment by erquhart** [comment]
Agreed, if anyone would still like to review, please open a new issue with change suggestions (or a PR with improvements).

**Comment by steffenz** [comment]
Awesome - glad I could help! Thanks for being so helpful and friendly, everybody. Hopefully, I'll find something else I can contribute to further down the road! 😁 Really love the work you do, rock on!

**Pull request merged**